packages/plugin/src/configs/schema-recommended.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * 🚨 IMPORTANT! Do not manually modify this file. Run: `yarn generate-configs`
+ */
+
+export default {
+  extends: ['plugin:@graphql-eslint/base'],
+  rules: {
+    '@graphql-eslint/description-style': 'error',
```

Comment:
Consider including `require-description` in recommended set. It doesn't require any parser options, and it's definitely a widely recommended practice.

Reply:
added